frontends/web/src/i18n/utils.ts

```diff
@@ -41,3 +41,22 @@ export const getRegionNameFromLocale = (nativeLocale: string): string => {
     return '';
   }
 };
+
+/**
+ * Workaround to add missing region info
+ * @param locale
+ * @returns locale-with-region
+ */
+export const getLocaleWithRegion = (locale: string) => {
+  if (locale.includes('-') || locale.includes('_')) {
+    return locale;
+  }
+  switch (locale) {
+  case 'en':
+    return 'en-US';
+  case 'de':
+    return 'de-DE';
+  default:
+    return `${locale}-${locale.toUpperCase()}`;
+  }
+};
```

Comment:
This is a bit silly and doesn't work properly. I'll just replace with a map of the actual languages they support.